components/PlantCard/styles.ts

```diff
@@ -0,0 +1,97 @@
+import styled from 'styled-components';
+
+export const Card = styled.div`
+  position: relative;
+  height: 40vh;
+  width: 20vw;
+  display: flex;
+  flex-direction: column;
+  align-items: start;
+  border-radius: 12px;
+  background-color: white;
+  box-shadow:
+    0 24px 38px 3px rgba(0, 0, 0, 0.14),
+    0 9px 46px 8px rgba(0, 0, 0, 0.12),
+    0 11px 15px -7px rgba(0, 0, 0, 0.2);
```

Comment:
honestly the shadow looks rlly cool, but we want the code to be consistent w the designs, which don't include the drop shadow. Kyrene said she's working on increasing the contrast between PlantCard and background, so it's ok to leave it for now, since we're still waiting for final designs, but just flagging that we might need to remove this later